README.md

```diff
@@ -93,11 +93,10 @@ packet-cloud-config   Opaque                                1         2m
 
 ##
```

Comment:
This looks like a missed title

Reply:
It is, but it was there before. Hold on, I will fix it.

Reply:
Fixed. Nice catch.